Lib/test/test_unittest/test_discovery.py

```diff
@@ -624,9 +628,9 @@ def test_command_line_handling_discover_by_default(self):
         program.parseArgs(['something'])
         self.assertEqual(args, [[]])
         self.assertEqual(program.verbosity, 1)
-        self.assertIs(program.buffer, False)
-        self.assertIs(program.catchbreak, False)
-        self.assertIs(program.failfast, False)
+        self.assertFalse(program.buffer)
+        self.assertFalse(program.catchbreak)
+        self.assertFalse(program.failfast)
```

Comment:
Note that these aren't testing precisely the same thing anymore: `self.assertFalse` just checks that the value is falsey, whereas `self.assertIs(x, False)` is much stricter: it checks that `x` actually is the singleton object `False`

Reply:
Also removed!